pyQuARC/schemas/rule_mapping.json

```diff
@@ -3720,7 +3720,7 @@
             "umm-c": [
                 {
                     "fields": [
-                        "ResourceCitation/OtherCitationDetails"
+                        "CollectionCitation/Title"
```

Comment:
Has this changed in the original schema as well?

Reply:
The "ResourceCitation/OtherCitationDetails" field appears to be from the umm-cmn-json schema. The "CollectionCitation/Title" field is from the umm-c-json schema. I believe the former was entered by mistake. Please let me know if there are additional concerns.